lib/mix/tasks/search.add.ex

```diff
@@ -3,7 +3,7 @@ defmodule Mix.Tasks.Search.Add do
   alias Search.HexClient
 
   @moduledoc """
-  Usage: mix #{Mix.Task.task_name(__MODULE__)} <PACKAGE> [<VERSION>]
+  Usage: mix #{Mix.Task.task_name(__MODULE__)} <PACKAGE> [version:<VERSION>] [max_size:<MAX_SIZE>]
```

Comment:
instead of `version:X`, `max_size:X`, lets' use `--version X`, `--max-size X`. See https://hexdocs.pm/elixir/OptionParser.html#parse!/2. Should fit right in here and you can remove parse_args/2`. This is an internal task so not a huge deal but if we can remove code, let's!

Reply:
Cool, I don't know how I didn't find this on my own; will replace